src/views/Recorder/EmptyState.tsx

```diff
@@ -32,9 +40,18 @@ const RecorderEmptyStateSchema = z.object({
 type RecorderEmptyStateFields = z.infer<typeof RecorderEmptyStateSchema>
 
 export function EmptyState({ isLoading, onStart }: EmptyStateProps) {
+  const { data: settings } = useSettings()
+
   const proxyStatus = useProxyStatus()
   const { data: isBrowserInstalled } = useBrowserCheck()
 
+  const [captureBrowser = true, setCaptureBrowser] = useLocalStorage(
```

Comment:
`useLocalStorage` returns `T | undefined` even if you pass an initial value. Doing `captureBrowser = true` is just pleasing the type checker.